spec/traits.dd

```diff
@@ -939,6 +940,9 @@ $(H2 $(LNAME2 specialkeywords, Special Keywords))
     $(P $(CODE __FILE__) and $(CODE __LINE__) expand to the source
```

Comment:
For `__FILE__` I think that a short sentence could explain that it _expands to the source file name, as passed to the compiler_.